umich_api/api_utils.py

```diff
@@ -100,22 +113,27 @@ def _api_call(self, api_call, method="GET", payload=None):
             self.__log.debug(resp.text)
             return resp
 
-    def get_pagination_query(self, headers):
-        return None
+    def get_next_page(self, response):
```

Comment:
would it be helpful to define a routine for get_total_page_number?

Reply:
I don't believe the total page number is a value that is available. (At least in Canvas) https://canvas.instructure.com/doc/api/file.pagination.html

The only way to determine that would be to iterate through the entire set of pages. 

I'm thinking as part of #10 to implement a method that will go through all of the pages and retrieve the full json list this could for sure contain a field to indicate how many pages it iterated through.